portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,13 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComment()">
```

Comment:
I think the onload attribute can be added to the `<div id="comments-container">` instead?

Reply:
Bumping this to make sure you see this